xgi/classes/hypergraphviews.py

```diff
@@ -100,6 +99,8 @@ def subhypergraph_view(H, filtered_nodes=None, filtered_edges=None):
     newH._edge_attr = {edge: H._edge_attr[edge] for edge in newH._edge}
 
     # Add the filtered nodes with connections to the remaining edges after filtering
-    newH._node = {node: set(H._node[node]).intersection(newH._edge.keys()) for node in nodes}
+    newH._node = {
+        node: set(H._node[node]).intersection(newH._edge.keys()) for node in nodes
+    }
```

Comment:
Suggestion: we should look into using [`black`](https://black.readthedocs.io/en/stable/) or some other similar tool.

Reply:
I am using black but possibly the wrong version?

Reply:
I see! What I meant is we should add a GH action that checks for black compliance on every PR.

Reply:
And we should be very careful whenever updating the version of black since that can mess up the codebase a bit.